### ISSUE TITLE
No such file or directory while unpacking and moving

### ISSUE THREAD

**Issue by dkingofpa** [applicationsonline/librarian#98]
Here's a snippet from the `--verbose` output.  The full output is [available here](https://gist.github.com/3417230).  The Cheffile I'm using is [available here](https://github.com/xforty/vagrant-drupal/blob/0.4.0/Cheffile).

```
[Librarian] Unpacking tmp/librarian/cache/source/chef/site/877777683730772c36b1e6a3fc3aa2c3/selinux/version-uri/700fe1613a13277556d04f778952724d/package.tar.gz to tmp/librarian/scratch/c115393f7af110d299b1cd18d6299745
[Librarian] Moving tmp/librarian/scratch/c115393f7af110d299b1cd18d6299745/selinux to tmp/librarian/cache/source/chef/site/877777683730772c36b1e6a3fc3aa2c3/selinux/version-uri/700fe1613a13277556d04f778952724d/package

/Users/dking/.rvm/rubies/ruby-1.9.2-p290/lib/ruby/1.9.1/fileutils.rb:1423:in `stat': No such file or directory - /Users/dking/Projects/test/tmp/librarian/scratch/c115393f7af110d299b1cd18d6299745/selinux (Errno::ENOENT)
    from /Users/dking/.rvm/rubies/ruby-1.9.2-p290/lib/ruby/1.9.1/fileutils.rb:1423:in `block in fu_each_src_dest'
    from /Users/dking/.rvm/rubies/ruby-1.9.2-p290/lib/ruby/1.9.1/fileutils.rb:1439:in `fu_each_src_dest0'
    from /Users/dking/.rvm/rubies/ruby-1.9.2-p290/lib/ruby/1.9.1/fileutils.rb:1421:in `fu_each_src_dest'
    from /Users/dking/.rvm/rubies/ruby-1.9.2-p290/lib/ruby/1.9.1/fileutils.rb:504:in `mv'
    from /Users/dking/.rvm/gems/ruby-1.9.2-p290/gems/librarian-0.0.24/lib/librarian/chef/source/site.rb:298:in `unpack_package!'
    from /Users/dking/.rvm/gems/ruby-1.9.2-p290/gems/librarian-0.0.24/lib/librarian/chef/source/site.rb:237:in `cache_version_uri_unpacked!'
    from /Users/dking/.rvm/gems/ruby-1.9.2-p290/gems/librarian-0.0.24/lib/librarian/chef/source/site.rb:100:in `version_uri_manifest'
    from /Users/dking/.rvm/gems/ruby-1.9.2-p290/gems/librarian-0.0.24/lib/librarian/chef/source/site.rb:94:in `version_manifest'
    from /Users/dking/.rvm/gems/ruby-1.9.2-p290/gems/librarian-0.0.24/lib/librarian/chef/source/site.rb:59:in `version_dependencies'
    from /Users/dking/.rvm/gems/ruby-1.9.2-p290/gems/librarian-0.0.24/lib/librarian/chef/source/site.rb:421:in `fetch_dependencies'
    from /Users/dking/.rvm/gems/ruby-1.9.2-p290/gems/librarian-0.0.24/lib/librarian/manifest.rb:121:in `fetch_dependencies!'
    from /Users/dking/.rvm/gems/ruby-1.9.2-p290/gems/librarian-0.0.24/lib/librarian/manifest.rb:113:in `fetched_dependencies'
    from /Users/dking/.rvm/gems/ruby-1.9.2-p290/gems/librarian-0.0.24/lib/librarian/manifest.rb:77:in `dependencies'
    from /Users/dking/.rvm/gems/ruby-1.9.2-p290/gems/librarian-0.0.24/lib/librarian/resolver/implementation.rb:71:in `block (4 levels) in recursive_resolve'
```

**Comment by yfeldblum** [comment]
Just recording a couple notes:
- The tarball for `selinux-0.1.0` is named `cookbook-selinux.tgz`.
- The directory in the tarball for `selinux-0.1.0` is named `cookbook-selinux`.
- The metadata for `selinux-0.1.0` has a cookbook name of `selinux`.

**Comment by yfeldblum** [comment]
Fixed in d61d9efd664609f07a388a46ed434c674e3cd618.
